decimal64.go

```diff
@@ -454,9 +453,32 @@ func (d Decimal64) IsInt() bool {
 		return false
 	}
 }
-func (d Decimal64) isZero() bool {
-	fl, _, _, significand := d.parts()
-	return significand == 0 && fl == flNormal
+
+// IsSubnormal returns true iff d is a subnormal.
+func (d Decimal64) IsSubnormal() bool {
+	flavor, _, exp, significand := d.parts()
+	return significand < decimal64Base && exp >= minExp && flavor == flNormal && significand != 0
```

Comment:
Put `significand != 0` at the front, so it's together with `significand < decimal64Base`.